pkg/v1/cli/command/core/repo.go

```diff
@@ -152,6 +155,7 @@ var updateRepoCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
+		log.Successf("Successfully updated repository configuration for %s", repoName)
```

Comment:
same here...start the message with a small letter "successfully" to be consistent with other messages.